src/main/webapp/app/account/settings/settings.route.ts

```diff
@@ -8,7 +8,7 @@ export const settingsRoute: Route = {
   path: 'settings',
   component: SettingsComponent,
   data: {
-    authorities: [Authority.USER],
+    authorities: [Authority.STUDENT, Authority.TEACHER],
```

Comment:
3 szerep

Reply:
de maradhat igy, ha most csak 2 szerepet tamogatunk

Reply:
A 3. szerepet is hozádatam